### PR TITLE
Added the possibility to disable distance based roughness for reflection probes.

### PR DESCRIPTION
**Pull request by anisunity**
Recently we changed the behavior of the reflection probes to have an approximation for the distance based BRDF integration (that we call distance based roughness). While the feature increases the image quality, it requires the user to be able to set up a proxy that matches roughly the geometry. That is not always possible (exteriors, oddly shapes rooms, etc.). So we added a toggle on the reflection probe itself to be able to disable the feature.
The checkbox is on by default.

Testing status
Ran the HDRP tests locally they were all green.
Compared with and without the checkbox and the behavior is the same as before the feature was added.

### CODE REVIEW

**Comment by TomasKiniulis** [comment]
Hey @anisunity! Anything in particular that QA should pay attention to testing this?

**Comment by anisunity** [comment]
I don't think so @TomasKiniulis, it is a quite simple PR (only couple lines)

**Review by iM0ve — CHANGES_REQUESTED** [review]
The feature is working but I have a few UX suggestions.

1) Since it only works when Proxies are used. Gray the Distance Based Roughness when Proxy is set to None and Use Influence Volume As Proxy Volume is unchecked.
2) Move the feature into Projection Settings if it makes sense.
3) Adjust the tooltip so it clearly states that will only have a positive result if Proxy closely matches the environment 
@JordanL8 tooltip suggestion:
"When enabled, HDRP uses the assigned Proxy Volume to calculate distance based roughness for reflections. This produces more physically-accurate results if the Proxy Volume closely matches the environment"

image

File: com.unity.render-pipelines.high-definition/Runtime/Utilities/ProbeSettings.cs [hunk 1]
```diff
@@ -278,6 +281,9 @@ public static ProbeSettings NewDefault()
         /// <summary>Defines if the planar reflection should support rough reflections.</summary>
         public bool roughReflections;
 
+        /// <summary>Defines if the reflection probe should support the distance based roughness.</summary>
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Runtime/Utilities/ProbeSettings.cs:284]: Indicates whether the ReflectionProbe supports distance-based roughness.

**Review by JordanL8 — CHANGES_REQUESTED** [review]
Can you add the small API summary change as well as the tooltip change Motiejus mentioned

**Comment by anisunity** [comment]
Did the review corrections and merged master and relaunching the tests

**Review by iM0ve — APPROVED** [review]
Issue 1 - will be solved later since its a more complicated UX matter. Tracked in fogbugz
Issue 2 - Fixed
Issue 3 - Fixed

3Nk4sR4YLX

**Review by sebastienlagarde — APPROVED** [review]

**Comment by sebastienlagarde** [comment]
Doc correction is done. Merging

**Pull request merged**